delegates.json

```diff
@@ -63,5 +63,10 @@
         "name": "Tensor.Exchange", 
         "url": "www.tensor.exchange", 
         "description": "Bittensor's first community OTC exchange"
+    },
+    "5Gpt8XWFTXmKrRF1qaxcBQLvnPLpKi6Pt2XC4vVQR7gqNKtU": {
+        "name": "StakeBoost Validator", 
+        "url": "https://taostats.io/validators/stakeboost-validator/", 
+        "description": "We refund 50% of the 18% take that validators get from delegated stake emissions. Payback is automatic after a minimum holding period of 30 days to the users coldkey."
```

Comment:
Hi, please don't include any "refunds", etc in the description

Reply:
why not? this is how we want to make it attractive for now.  we have a mission beyond that in graphics AI. but for now we need to have a USP, because we are on the edge with our stake and if we get out of top128 we cant contribute anything

Reply:
> why not  

We have a strict rule not to include these kinds of statements in our codebase. Apologies